src/components/ColorPicker.tsx

```diff
@@ -270,26 +271,38 @@ export const ColorPicker = ({
       </div>
       <React.Suspense fallback="">
         {isActive ? (
-          <Popover
-            onCloseRequest={(event) =>
-              event.target !== pickerButton.current && setActive(false)
-            }
-          >
-            <Picker
-              colors={colors[type]}
-              color={color || null}
-              onChange={(changedColor) => {
-                onChange(changedColor);
-              }}
-              onClose={() => {
-                setActive(false);
-                pickerButton.current?.focus();
-              }}
-              label={label}
-              showInput={false}
-              type={type}
-            />
-          </Popover>
+          <PopoverModal>
+            <Popover
+              onCloseRequest={(event) =>
+                event.target !== pickerButton.current && setActive(false)
+              }
+              top={
+                type === "canvasBackground"
+                  ? 102
+                  : type === "elementStroke"
+                  ? 190
+                  : type === "elementBackground"
+                  ? 250
```

Comment:
We'll want to calculate these dynamically (using `getBoundingClientRect()` or similar) instead of hardcoding them. Otherwise this won't work when the properties panel is scrolled and similar:

![image](https://user-images.githubusercontent.com/5153846/126390656-01a459ea-dbe9-4f34-be60-a476999da209.png)

Reply:
Got it. I work on it